tests/phpunit/tests/post.php

```diff
@@ -956,100 +967,102 @@ public function test_wp_resolve_post_date_regex( $date, $expected ) {
 	}
 
 	/**
-	 * @ticket 26798
+	 * Data provider for test_wp_resolve_post_date_regex().
+	 *
+	 * @return array<array{ date: string, expected: string|false }>
```

Comment:
Ditto above:


```suggestion
	 * @return array<string, array{ date: string, expected: string|false }>
```

But this would require coming up with names for all of the tests to use in the keys.